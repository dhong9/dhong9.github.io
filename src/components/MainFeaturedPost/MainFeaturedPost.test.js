// React testing libraries
import renderer from "react-test-renderer";

// Component to test
import MainFeaturedPost from "components/MainFeaturedPost";

describe("MainFeaturedPost", () => {
  it("renders", () => {
    const posts = [{
        postName: "Bicycles",
        body: "It's healthy to bike to work everyday."
    }];
    const component = renderer.create(<MainFeaturedPost posts={posts} />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
