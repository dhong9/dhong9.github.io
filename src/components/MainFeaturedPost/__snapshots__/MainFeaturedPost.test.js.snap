// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainFeaturedPost renders 1`] = `
<div
  className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-rl1oml-MuiPaper-root"
>
  <img
    alt="main image description"
    src="https://source.unsplash.com/random"
    style={
      {
        "display": "none",
      }
    }
  />
  <div
    className="MuiBox-root css-1plq5y9"
  />
  <div
    className="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-md-6 css-1k7sk4d-MuiGrid-root"
    >
      <div
        className="MuiBox-root css-30n99s"
      >
        <h1
          className="MuiTypography-root MuiTypography-h3 MuiTypography-gutterBottom css-1007tex-MuiTypography-root"
        />
        <p
          className="MuiTypography-root MuiTypography-h5 MuiTypography-paragraph css-187jqjf-MuiTypography-root"
        />
        <a
          className="MuiTypography-root MuiTypography-subtitle1 MuiLink-root MuiLink-underlineAlways css-1f9rlei-MuiTypography-root-MuiLink-root"
          href="#"
          onBlur={[Function]}
          onFocus={[Function]}
        >
          Continue reading...
        </a>
      </div>
    </div>
  </div>
</div>
`;
